### PR TITLE
Bookings Contact Information - Back End and Front End

### PR DESCRIPTION
**Pull request by akroon3r**

### CODE REVIEW

File: frontend/src/booking/edit-booking-modal.vue [hunk 1]
```diff
@@ -57,7 +57,7 @@
           <b-form-row v-if="!examAssociated">
             <b-col>
               <b-form-group>
-                <label :style="{color: labelColor}">Event Title</label><br>
+                <label :style="{color: labelColor}">Scheduling Party</label><br>
```
Comment (sjrumsby) [inline frontend/src/booking/edit-booking-modal.vue]: Should this stay as "Event Title" when the it's an exam being booked?

**Comment by sjrumsby** [comment]
This will also need to have the migration updated, since another migration has been added in your other PR.

File: frontend/src/booking/edit-booking-modal.vue (hunk 1 above)
Comment (akroon3r) [reply frontend/src/booking/edit-booking-modal.vue]: I thought that Becky said that this was a blanket change?

Comment (akroon3r) [reply frontend/src/booking/edit-booking-modal.vue]: https://trello.com/c/2JvqBVuD/293-editing-the-non-exam-booking-also-update-the-event-title-field-to-be-scheduling-party

**Comment by akroon3r** [comment]
Fixed the alembic issues and pushed up.

File: frontend/src/booking/edit-booking-modal.vue (hunk 1 above)
Comment (sjrumsby) [reply frontend/src/booking/edit-booking-modal.vue]: It says for "non-exam booking", but I can see how that can be ambiguous. Can you follow up with Becky to confirm?

Comment (akroon3r) [reply frontend/src/booking/edit-booking-modal.vue]: Yes.

**Pull request merged**